doc/mgr/zabbix.rst

```diff
@@ -47,8 +47,8 @@ Configuration
 
 Two configuration keys are mandatory for the module to work:
 
-- mgr/zabbix/zabbix_host
-- mgr/zabbix/identifier
+- zabbix_host
+- identifier
```

Comment:
[ot],

@jcsp @wido   the settings are stored using the prefix of `get_mgr_id() / ` (i.e. `$name/`), so there is chance that the config item name of one plugin could collide with those in another plugin. shall we update `{get,set}{_localized,}_config()` to address this by, for example, append the plugin name to the prefix?